src/chaosinventory/authentication/urls.py

```diff
@@ -6,8 +6,8 @@
 )
 
 urlpatterns = [
-    path('token', AuthTokenView.as_view()),
+    path('token/', AuthTokenView.as_view()),
     path('token/credentials', ObtainAuthTokenWithCredentialsView.as_view()),
```

Comment:
Personally, I would say, every url should have a / at the end.

Reply:
Django should somehwat do this (however it didn't do that for the `token/` in my testing...), in combination with #82 and the significant refactor of this in #78 we should fix it over there.